### PR TITLE
manifest: sdk-zephyr: Update nrf hw model revision

### PR DESCRIPTION
**Pull request by ilhanates**
sdk-zephyr revision is updated to point latest nrf_hw_models

### CODE REVIEW

**Comment by ilhanates** [comment]
Tests [fail ](https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsdk-nrf/detail/PR-12082/3/pipeline) due to compilation issues possibly related to a previous zephyr [commit ](https://github.com/nrfconnect/sdk-zephyr/commit/9e7d6bbdf82f8d1fa17c1dc82c6a868554000025) @ankuns Do you have any idea to fix this?

https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsdk-nrf/detail/PR-12082/3/pipeline
`[2023-08-21T14:44:44.909Z] ../../../../../nrf/samples/peripheral/802154_phy_test/src/rf_proc.c:286:6: error: conflicting types for 'nrf_802154_energy_detected'; have 'void(uint8_t)' {aka 'void(unsigned char)'}
[2023-08-21T14:44:44.909Z]   286 | void nrf_802154_energy_detected(uint8_t result)
[2023-08-21T14:44:44.909Z]       |      ^~~~~~~~~~~~~~~~~~~~~~~~~~
[2023-08-21T14:44:44.909Z] In file included from ../../../../../nrfxlib/nrf_802154/common/include/nrf_802154.h:47,
[2023-08-21T14:44:44.909Z]                  from ../../../../../nrf/samples/peripheral/802154_phy_test/src/rf_proc.c:24:
[2023-08-21T14:44:44.909Z] ../../../../../nrfxlib/nrf_802154/common/include/nrf_802154_callouts.h:77:13: note: previous declaration of 'nrf_802154_energy_detected' with type 'void(const nrf_802154_energy_detected_t *)'
[2023-08-21T14:44:44.909Z]    77 | extern void nrf_802154_energy_detected(const nrf_802154_energy_detected_t * p_result);
[2023-08-21T14:44:44.909Z]       |             ^~~~~~~~~~~~~~~~~~~~~~~~~~
[2023-08-21T14:44:44.909Z] [19/295] Building C object CMakeFiles/app.dir/src/ptt_conf.c.obj`

**Comment by ankuns** [comment]
This PR must be merged first first https://github.com/nrfconnect/sdk-nrf/pull/11767

File: west.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 9e7d6bbdf82f8d1fa17c1dc82c6a868554000025
+      revision: pull/1284/head
```
Comment (rlubos) [inline west.yml:62]: Please update SHA: `https://github.com/zephyrproject-rtos/zephyr/pull/61353`

Comment (ilhanates) [reply west.yml]: Done

**Review by laxiLang — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**